tests/plugins/test_pin_operator_digests.py

```diff
@@ -294,15 +329,15 @@ def test_orchestrator(self, docker_tasker, tmpdir, caplog):
             'To be replaced:\n'
             'keep-registry/ns/foo:latest -> keep-registry/ns/foo@sha256:abcdef\n'
             'keep-registry/ns/spam@sha256:123456 - no change\n'
-            'replace-registry/ns/bar:1 -> new-registry/ns/bar@sha256:fedcba\n'
-            'replace-registry/ns/eggs@sha256:654321 -> new-registry/ns/eggs@sha256:654321'
+            'replace-registry/ns/bar:1 -> new-registry/ns/baz@sha256:fedcba\n'
```

Comment:
Maybe `replace-registry/ns/bar:1 -> new-registry/baz@sha256:fedcba` should be included too